### ISSUE TITLE
Appointment Dates and Times Not Updating Correctly

### ISSUE THREAD

**Issue by robbie-Benson** [AnthonyKon/Tafe19S101#1]
Currently within the Appointments page, when the date is updated it seems to default to "01/01/2000" and when either the start time or end time are updated they default to a value like "0:0.00000015"

**Comment by JarradStawiarski** [comment]
What’s your discord? I’ll invite you in and screenshot my code for dates, which works.

Sent from my iPhone

On 1 Apr 2019, at 7:59 pm, robbie-Benson <notifications@github.com<mailto:notifications@github.com>> wrote:


Currently within the Appointments page, when the date is updated it seems to default to "01/01/2000" and when either the start time or end time are updated they default to a value like "0:0.00000015"

**Comment by robbie-Benson** [comment]
My discord is Wis#9513

**Comment by JarradStawiarski** [comment]
That didn’t work for some reason, the TAFE discord i made is https://discord.gg/AsVtBh

Sent from my iPhone

On 1 Apr 2019, at 9:35 pm, robbie-Benson <notifications@github.com<mailto:notifications@github.com>> wrote:

Wis#9513

**Comment by robbie-Benson** [comment]
This issue was solved by changing the dates and times in the database from 'DateTime' and 'TimeSpan' to 'string'. Each individual part of the date or time is concatenated into a finalDate or finalTime before being inserted into the database.

**Comment by JarradStawiarski** [comment]
Yeah need to parse all DateTime entries into strings otherwise it doesn't work
